tests/foreman/api/test_http_proxy.py

```diff
@@ -125,3 +126,75 @@ def test_positive_end_to_end(setup_http_proxy, module_target_sat, module_manifes
     )
     assert len(yum_repo['output']) >= 1
     assert 'quay/busybox' in yum_repo['output']
+
+
+@pytest.mark.upgrade
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.run_in_one_thread
+@pytest.mark.parametrize(
+    'setup_http_proxy',
+    [None, True, False],
+    indirect=True,
+    ids=['no_http_proxy', 'auth_http_proxy', 'unauth_http_proxy'],
+)
+@pytest.mark.tier3
+def test_positive_auto_attach(
```

Comment:
Non blocking - test name can be more descriptive `test_positive_auto_attach_subscription(s)`

Reply:
OR `test_positive_auto_attach_with_proxy()`